### ISSUE TITLE
about model weights

### ISSUE THREAD

**Issue by XingweiChen** [TJCVRS/CRNN_Tensorflow#78]
Hello,

I had tried to run your demo code, but I found that "shadownet_2017-09-29-19-16-33.ckpt-39999" is no longer there.
I had also tried shadownet_2017-10-17-11-47-46.ckpt-199999.data-00000-of-00001, but still doesn't work.

DataLossError (see above for traceback): Unable to open table file model/shadownet/shadownet_2017-10-17-11-47-46.ckpt-199999.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?

Can you upload the "shadownet_2017-09-29-19-16-33.ckpt-39999" again? Thank you very much.

It will be better if you can send me the training dataset or told me where can I download it. Thank you.
(auchenxingwei@gmail.com)
Best,
brook

**Comment by sniper0110** [comment]
Hello,
I am having the same problem, can you please upload the "shadownet_2017-09-29-19-16-33.ckpt-39999" model again please?

**Comment by mdbenito** [comment]
In order to use the saved models you need to roll back to a version of the code using Tensorflow 1.3. Search among the closed issues for pointers

**Comment by harshalcse** [comment]
Hello ,
Even I am having same problem, can you please upload "shadownet_2017-09-29-19-16-33.ckpt-39999" model again please?
Mail ID : harshalcse2910gmail.com
@XingweiChen @MaybeShewill-CV @mdbenito @sniper0110 please help.
